helm-chart/renku/templates/notebooks/role.yaml

```diff
@@ -67,5 +67,17 @@ rules:
       - list
       - get
       - watch
+  - apiGroups:
+      - amalthea.dev
+    resources:
+      - amaltheasessions
```

Comment:
Is it gonna be okay to create a role that has permissions on a resource that doesn't exist (yet)?

Reply:
I will try this on dev with a madeup CR. One sec.

Reply:
Yes I just made a role on dev with a completely made up CR that does not exist. It was successfully created.